src/mon/ConnectionTracker.h

```diff
@@ -185,6 +185,18 @@ class ConnectionTracker {
     rank = new_rank;
     my_reports.rank = rank;
   }
+  /**
+  * Check if our ranks are clean.
+  * In the future we also want to check the reports
+  * current and history of each peer_report.
+  */
+  bool is_clean(int mon_rank) {
+    if (rank != mon_rank ||
+      my_reports.rank != mon_rank) {
+      return 0;
```

Comment:
Nit: the function is declared as a bool; we should return a bool instead of relying on ints being cast.

Reply:
Addressed in new push